gravity.sh

```diff
@@ -847,6 +847,45 @@ gravity_Cleanup() {
   fi
 }
 
+database_recovery() {
+  str="Checking integrity of existing gravity database"
+  echo -ne "  ${INFO} ${str}..."
+  if result="$(pihole-FTL sqlite3 "${gravityDBfile}" "PRAGMA integrity_check" 2>&1)"; then
+    echo -e "${OVER}  ${TICK} ${str} - no errors found"
+
+    str="Checking foreign keys of existing gravity database"
+    echo -ne "  ${INFO} ${str}..."
+    if result="$(pihole-FTL sqlite3 "${gravityDBfile}" "PRAGMA foreign_key_check" 2>&1)"; then
+      echo -e "${OVER}  ${TICK} ${str} - no errors found"
+      if [[ $1 != "force" ]]; then
```

Comment:
What is this checking for? I can't see anywhere that the word `force` gets passed through to this function? I also tried manually typing it (`pihole -g -r recover force`) but it didn't seem to do anything?